#include <iostream>

int main(){

    
/*
    
    
    for (size_t i = 0; i < 1000; i++)
    {
        try
    {
        int * data = new int[100000000000];
       std::cout<<"Program ending well"<<std::endl;
    }
    catch(const std::exception& ex)
    {
        std::cout<<"Exception: "<<ex.what()<<std::endl;
    }
    
    }*/


    for (size_t i = 0; i < 1000; i++)
    {
     
        int * data = new int[100000000000];
      
    }
    
    
    
    
return 0;

}